_posts/2017-10-23-survey-focus-group-update.md

```diff
@@ -0,0 +1,75 @@
+
+The focus group’s first [meeting](https://docs.google.com/document/d/16GEJiHNSla0aE1oCRjqDRyLlIz1q2NCXFJt3vWs70zw/edit) was Friday, October 20th 2017. The primary goal of this meeting was to determine how we could assign unique identifiers to our learners and still have their data remain confidential. Additionally, we discussed the need to collect demographic information, and how to accomplish that in an inclusive way.
+
+In terms of unique identifiers, an idea was to ask learners to provide their personal e-mail address as their unique identifier. Issues were raised around learners not remembering which e-mail address they gave, and that by providing their e-mail address their responses would not be anonymous. We discussed stating in the introductory language that responses would be *confidential*, but not *anonymous* as research staff (Kari Jordan and Erin Becker) have access to the data. There was also the idea of having SurveyMonkey create unique survey links for participants, but this would not be feasible because our survey links are embedded into the [workshop website template](https://github.com/datacarpentry/workshop-template). SurveyMonkey does offer the ability to create [custom variables](https://help.surveymonkey.com/articles/en_US/kb/What-are-custom-variables-and-how-do-I-use-them) however, this will add more manual work for staff and workshop hosts. A graphic was also discussed as an option for unique identifiers, however, we are concerned that learners may not remember what graphic they chose. We are still searching for a solution, so if you have ideas please [get in touch](mailto:kariljordan@carpentries.org). 
+
+U.S. based workshop participants are asked demographic questions (race/ethnicity, gender, etc.) Collecting this type of information can be challenging, and we want to be sure we are asking demographic questions in an inclusive manner. Thus, instead of asking *what is your gender*, and having learners select from the following, 
+
+          + Female 
+          + Male 
+          + Other 
+          + Prefer not to say 
+
+the *gender* question was changed to a free response question where learners are able to type in how they identify themselves. This will result in more data cleaning, but this can easily be done in OpenRefine.
+
+Additionally, instead of asking learners to check a box identifying whether or not they are dealing with a disability, we added the following questions:
+
+        **Accessibility** Do you need support for any accessibility requirements that would improve your experience at the      
+        conference? If so, please describe.  
+
+        **Accommodation. Check all that apply.**
```

Comment:
We need to keep in mind that not all of our workshop venues will be able to make all accommodations. For example, providing translation for every language is not feasible. Also, these survey responses go to the workshop instructors, not the host, and the instructors are often not local and would not know how to arrange for these accommodations.

Reply:
If this is information we aren't interested in collecting, we will not ask these questions. Instead, we will ask learners to contact their workshop host if they have accessibility or other accommodation needs.